### DISCUSSION TITLE
Web Login

### DISCUSSION THREAD

**Discussion by GustavPCG** [ej52/proxmox-scripts#112] [Q&A]
What credentials do I use to first login to the web ui?
Thx in advance!

**Comment by cdonahue55** [comment]
try: admin@example.com
password: changeme

**Comment by Eeems** [comment]
To expand on @cdonahue55's answer, the default user is documented in the upstream nginx-proxy-manager readme: https://github.com/NginxProxyManager/nginx-proxy-manager#quick-setup
